2016/03.md

```diff
@@ -79,6 +79,6 @@
 | Dates                    | Location          | Host       |
 |--------------------------|-------------------|------------|
 | 2016-05-23 to 2016-05-25 | Munich, DE        | Google     |
-| 2016-07-26 to 2016-07-28 | Remond, WA        | Microsoft  |
+| 2016-07-26 to 2016-07-28 | Remdond, WA      | Microsoft  |
```

Comment:
If by "correctly" you mean "incorrectly", then "Remdond" is right on! ;-)
https://github.com/tc39/agendas/pull/180